src/data/datasets.json

```diff
@@ -0,0 +1,145 @@
+{
+  "dataset" : [
+    {
+      "from" : "FROM http://stad.gent/dekrook/uitleningen-1996",
```

Comment:
Ik zou die 'FROM' er uit laten en pas zelf bijplakken wanneer je de dataset gebruikt in een query.
Dan is je data iets netter.

Overweeg ook om overal een 'id' toe te voegen.